specification/README.md

```diff
@@ -1096,6 +1095,22 @@ The following example colors features with a height above 90 as red and the othe
 
 For complete details, see the [Declarative Styling](./Styling/) specification.
 
+
+## Version History
+
+- [3D Tiles 1.0](https://github.com/CesiumGS/3d-tiles/tree/1.0) _(2018-12-14)_, approved as the [3D Tiles Specification 1.0](http://docs.opengeospatial.org/cs/18-053r2/18-053r2.html) by the Open Geospatial Consortium  
```

Comment:
```suggestion
- [3D Tiles 1.0](https://github.com/CesiumGS/3d-tiles/tree/1.0) _(2018-12-14)_, approved as an [OGC Community Standard](http://docs.opengeospatial.org/cs/18-053r2/18-053r2.html)
```

Reply:
Actually this suggestion isn't great because it's not obvious that `OGC Community Standard` links to the 3D Tiles document. But I think it's important to have the term "Community Standard" somewhere in here.